build/less/mixins/mixins.less

```diff
@@ -0,0 +1,16 @@
+@import "vertical-rhythm";
+
+/* Micro ClearFix Mixin */
+.clearfix() {
```

Comment:
I would also avoid using .clearfix, as it ALWAYS leads to issues with Safari, especially if our layouts a built more with flexbox and css grid. Having these also makes styling harder, as they sometimes influence :first-child and :last-child